src/foam/nanos/crunch/Capability.js

```diff
@@ -348,7 +346,7 @@ foam.CLASS({
       ],
       documentation: `Checks if a permission or capability string is implied by the current capability`,
       javaCode: `
-        if ( ! this.getEnabled() ) return false;
+        if ( getLifecycleState() == LifecycleState.DELETED || getLifecycleState() == LifecycleState.REJECTED ) return false;
```

Comment:
why not getlifecyclestate != active ? does pending also mean enabled?

Reply:
Pending doesn't have much of a use case here - getlifecyclestate != active works fine here